hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java

```diff
@@ -342,8 +343,11 @@ private void checkThrottles(
         final var hederaConfig = configuration.getConfigData(HederaConfig.class);
         final var hooksConfig = configuration.getConfigData(HooksConfig.class);
         assertThrottlingPreconditions(txInfo, hederaConfig, hooksConfig);
-        if (hederaConfig.ingestThrottleEnabled()
-                && synchronizedThrottleAccumulator.shouldThrottle(txInfo, state, throttleUsages)) {
+        ThrottleResult throttleResult = synchronizedThrottleAccumulator.shouldThrottle(txInfo, state, throttleUsages);
+        if (throttleResult.hasValidationError()) {
+            throw new PreCheckException(throttleResult.validationError());
+        }
```

Comment:
This code should be under `hederaConfig.ingestThrottleEnabled()` this condition.